core/workflows/workflow_arxiv_metadata.py

```
@@ -0,0 +1,579 @@
+                    task="retrieval.passage"
+                )
+
+                # Flatten and associate with records
+                for record, record_chunks in zip(records_with_abstracts, all_chunks):
+                    for chunk in record_chunks:
+                        chunks_with_embeddings.append({
+                            'record': record,
+                            'chunk': chunk
+                        })
+
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)